clang/include/clang/Driver/CC1Options.td

```diff
@@ -899,6 +899,8 @@ def fsycl_int_header : Separate<["-"], "fsycl-int-header">,
   HelpText<"Generate SYCL integration header into this file.">;
 def fsycl_int_header_EQ : Joined<["-"], "fsycl-int-header=">,
   Alias<fsycl_int_header>;
+def fsycl_std_layout_requirement: Flag<["-"], "fsycl-std-layout-requirement">,
```

Comment:
How about more specific
```suggestion
def fsycl_std_layout_kernel_params: Flag<["-"], "fsycl-std-layout-kernel-params">,
```
?

Reply:
Sure, why not. I think language option should be renamed as well.

Reply:
Done.